doc/source/installation.rst

```diff
@@ -1,79 +1,89 @@
-.. |br| raw:: html
-
-  <br />
-
 Installation
 ============
 
-It is best to install the pycompwa framework within a virtual environment, so
-that all dependencies of pycompwa are contained within there. We recommend to
-use `Anaconda <https://www.anaconda.com/distribution/>`_ in combination with
-`Conda-Forge <https://conda-forge.org/>`_.
+Prerequisites
+^^^^^^^^^^^^^
 
-After you have `installed Anaconda
-<https://docs.anaconda.com/anaconda/install/>`_, set Conda-Forge as the
-default channel as follows:
+`pycompwa` is `available as a PyPI package
+<https://pypi.org/project/pycompwa/>`_. However, since the `pycompwa.ui` module
+contains Python bindings to the `ComPWA backend
+<https://github.com/ComPWA/ComPWA>`_, you first need to install certain C++
+prerequisites.
 
-.. code-block:: shell
+What you definitely need to install, is:
 
-  conda config --add channels conda-forge
-  conda config --set channel_priority strict
+* a build tool: `cmake <https://cmake.org/>`_
+* a C++ compiler:
+  `gcc <https://gcc.gnu.org/>`_ or `clang <https://clang.llvm.org/>`_
+* `Boost <https://www.boost.org/>`_
+* `Python3 <https://www.python.org/downloads/>`_
 
-Now, create a new environment with `all required dependencies
-<https://github.com/ComPWA/pycompwa/blob/master/requirements.txt>`_
-installed:
+The following package are optional:
 
-.. code-block:: shell
+* `ROOT <https://root.cern.ch/downloading-root>`_ and/or `Minuit2
+  <http://seal.web.cern.ch/seal/snapshot/work-packages/mathlibs/minuit/>`_
```

Comment:
I think we should clarify that at this point not having ROOT with Minuit2 does not allow any fitting. So even though its optional for building, the functionality without will be limited

Reply:
https://github.com/ComPWA/pycompwa/pull/83/commits/39981ea70f4bfe626d0ac0e92c29d3a854d6a601